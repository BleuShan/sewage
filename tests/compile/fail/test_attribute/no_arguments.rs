#[sewage::test]
fn run() {}

fn main() {}
