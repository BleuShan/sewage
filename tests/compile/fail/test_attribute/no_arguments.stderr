error: Missing arguments
 --> $DIR/no_arguments.rs:1:1
  |
1 | #[sewage::test]
  | ^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
